packages/core/src/Utils/Datadog/index.ts

```diff
@@ -0,0 +1,24 @@
+import { datadogRum } from '@datadog/browser-rum';
+
+const DATADOG_APP_ID = process.env.DATADOG_APPLICATION_ID ? process.env.DATADOG_APPLICATION_ID : '';
+const DATADOG_CLIENT_TOKEN = process.env.DATADOG_CLIENT_TOKEN ? process.env.DATADOG_CLIENT_TOKEN : '';
+const DATADOG_SESSION_SAMPLE_RATE = process.env.DATADOG_SESSION_SAMPLE_RATE
+    ? process.env.DATADOG_SESSION_SAMPLE_RATE
+    : 10;
+const DATADOG_SESSION_REPLAY_SAMPLE_RATE = process.env.DATADOG_SESSION_REPLAY_SAMPLE_RATE
+    ? process.env.DATADOG_SESSION_REPLAY_SAMPLE_RATE
+    : 1;
+
+datadogRum.init({
+    applicationId: DATADOG_APP_ID,
+    clientToken: DATADOG_CLIENT_TOKEN,
+    site: 'datadoghq.com',
+    service: 'deriv.com',
```

Comment:
Will it prevent it from running on test links and staging?

Reply:
it depends on the values. since we dont have the tokens on vercel it wont work there.
but it seems that we need it on staging. I see that @habib-deriv enabled it on staging just today